test/ctr.bats

```diff
@@ -108,3 +108,36 @@ function teardown() {
 	cleanup_pods
 	stop_ocid
 }
+
+@test "ctr status #127" {
```

Comment:
@mrunalp thought it would be nice to have references to issue fixed by a particular regression test